src/main/scala/interplay/PlayBuildBase.scala

```diff
@@ -236,6 +239,17 @@ object PlayReleaseBase extends AutoPlugin {
 
 }
 
+object PlayLicensePlugin extends AutoPlugin {
+
+  override def requires: Plugins = WhiteSourcePlugin
+
+  override def trigger: PluginTrigger = allRequirements
+
+  override lazy val projectSettings = Seq(
+    whitesourceProduct := "Lightbend Reactive Platform"
+  )
```

Comment:
Maybe we should add something in the readme about how to use this on a project. I think we also need the project name and token as described here: https://github.com/typesafehub/sbt-whitesource#setup